src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -10,6 +12,7 @@
 import org.bukkit.Material;
 import org.bukkit.NamespacedKey;
 
+
```

Comment:
Why was there another line added here

Reply:
I fixed that in the commit after that, as well as removing the useless import